pom.xml

```diff
@@ -97,13 +97,13 @@
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
     <!-- dependencies -->
-    <bookkeeper.version>4.3.7-TWTTR-OSS</bookkeeper.version>
+    <bookkeeper.version>4.5.0-SNAPSHOT</bookkeeper.version>
```

Comment:
@zhaijack since 4.5.0 is already released, I think we can remove `-SNAPSHOT` to use the released version.